src/cljc/athens/common_events/resolver/undo.cljc

```diff
@@ -0,0 +1,43 @@
+(ns athens.common-events.resolver.undo
+  (:require
+    [athens.common-db                     :as common-db]
+    [athens.common-events                 :as common-events]
+    [athens.common-events.graph.composite :as composite]
+    [athens.common-events.graph.ops       :as graph-ops]
+    [athens.common.logging                :as log]))
+
+
+(defn undo?
+  [event]
+  (-> event :event/op :op/trigger :op/undo))
+
+
+(defmulti resolve-atomic-op-to-undo-op
```

Comment:
We need to impl these operations, under the rules described in the doc https://github.com/athensresearch/athens/blob/main/doc/adr/0021-undo-redo.md#approach